src/main/java/net/snowflake/ingest/utils/SFException.java

```diff
@@ -58,4 +60,8 @@ public Object[] getParams() {
   public Throwable getCause() {
     return cause;
   }
+
+  public ErrorCode getErrorCode() {
```

Comment:
can you add a comment here saying this is error code returned from snowflake and nothing we created here?